src/newgrf_roadstop.cpp

```diff
@@ -171,6 +171,22 @@ uint32 RoadStopScopeResolver::GetVariable(byte variable, uint32 parameter, bool
 			return ssl.grfid;
 		}
 
+		/* 16 bit road stop ID of nearby tiles */
```

Comment:
Is the "16 bit" part important? As we don't mention how many bits for any of the other?